package.json

```diff
@@ -30,8 +30,8 @@
     "migration:revert": "yarn typeorm migration:revert"
   },
   "dependencies": {
-    "@aws-sdk/client-kms": "^3.687.0",
-    "@aws-sdk/client-s3": "^3.685.0",
+    "@aws-sdk/client-kms": "^3.699.0",
+    "@aws-sdk/client-s3": "^3.701.0",
```

Comment:
Installing both packages [is required](https://github.com/m-radzikowski/aws-sdk-client-mock/issues/197) to make the types work.